scripts/calculateRevenue.ts

```diff
@@ -63,14 +68,14 @@ function parseArgs() {
     .option('start-timestamp', {
       alias: 's',
       description:
-        'timestamp at which to start checking for revenue (since epoch)',
+        'timestamp at which to start checking for revenue (milliseconds since epoch)',
```

Comment:
maybe easier to do iso ts? or actually anything Date(..) can accept? E.g., if we don't care about times, can we just pass the date in `yyyy-mm-dd`?